active/SSTI.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * Made with ❤️ by Astra Security (https://www.getastra.com/)
+ * @author: Karthik UJ (karthik.uj@getastra.com)
+ * Version: 1.0
+ */
+
+// Labs: https://portswigger.net/web-security/all-labs#server-side-template-injection
+// More simple web apps made with SSTI vulnerable template engines: https://github.com/DiogoMRSilva/websitesVulnerableToSSTI
+
+// Import Logger
+var LoggerManager = Java.type("org.apache.logging.log4j.LogManager");
+var log = LoggerManager.getLogger("SSTI");
+
+// HasMap for alertTags
+var HashMap = Java.type('java.util.HashMap');
+
+function logger() {
+	print("[" + this["zap.script.name"] + "] " + arguments[0]);
+	log.debug("[" + this["zap.script.name"] + "] " + arguments[0]);
+}
+
+/**
+ * Scans a specific parameter in an HTTP message.
+ * The scan function will typically be called for every parameter in every URL and Form for every page.
+ * 
+ * @param as - the ActiveScan parent object that will do all the core interface tasks 
+ *     (i.e.: sending and receiving messages, providing access to Strength and Threshold settings,
+ *     raising alerts, etc.). This is an ScriptsActiveScanner object.
+ * @param msg - the HTTP Message being scanned. This is an HttpMessage object.
+ * @param {string} param - the name of the parameter being manipulated for this test/scan.
+ * @param {string} value - the original parameter value.
+ */
+function scan(as, msg, param, value) {
+	// Debugging can be done using println like this
```

Comment:
`println` is not being used here. This comment can be removed.

Reply:
removed ✔